C++/Examples/AccelGyroMag/AccelGyroMag.cpp

```diff
@@ -47,36 +47,50 @@ void print_help()
     printf("Sensors names: mpu is MPU9250, lsm is LSM9DS1\nFor help: -h\n");
 }
 
+std::string get_sensor_name(int argc, char *argv[])
+{
+    if (get_navio_version() == NAVIO2) {
+
+        if (argc < 2) {
+            printf("Enter parameter\n");
+            print_help();
+            return "-1";
+        }
+
+        // prevent the error message
+        opterr = 0;
+        int parameter;
+
+        while ((parameter = getopt(argc, argv, "i:h")) != -1) {
+            switch (parameter) {
+            case 'i': if (!strcmp(optarg,"mpu") ) return "mpu";
+                            else return "lsm";
+            case 'h': print_help(); return "-1";
+            case '?': printf("Wrong parameter.\n");
+                      print_help();
+                      return "-1";
```

Comment:
-1 is extremely bad! Please, say at least "none" or just use plain C strings and return a nullptr.